PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/ui/details/DetailsFragment.java

```diff
@@ -0,0 +1,42 @@
+package edu.cascadia.mobas.photopoints.ui.details;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+
+import com.google.android.material.textfield.TextInputEditText;
+
+import edu.cascadia.mobas.photopoints.R;
+import edu.cascadia.mobas.photopoints.model.Plant;
+
+public class DetailsFragment extends Fragment {
+    String text1;
```

Comment:
I know dummy data is not part of the DoD, it would be a nice to have to test the page. However, adding variables that don't do anything and that we have to remove afterwards is kind of useless. 

You can add private reference types to the TextViews and use findViewById to populate these in the onCreate.